src/App.vue

```diff
@@ -2,12 +2,13 @@
   <v-app style="background: linear-gradient(115.51deg, #00243E 0%, #003E3A 100%);" id="app">
       <Toolbar></Toolbar>
       <!-- TODO: remove :key similar to https://github.com/whynotearth/shinta-mani-wild/pull/298 -->
-      <router-view :key="$route.name + ($route.params.id || '')"/> 
+      <router-view :key="$route.name + ($route.params.id || '')"/>
   </v-app>
 </template>
 
 <script>
 const Toolbar = () => import ('@/components/Toolbar.vue')
+const ResetPassword = () => import ('@/components/Auth/ResetPassword.vue')
```

Comment:
We don't need this here. Please check the routes.js file and add a new route there.

Reply:
We have a `views` folder. every new route should be connected to a new component in the view folder, in that component we can import other components (reset-password component for example.).
So in views folder create a component: `ResetPasswordPage.vue` and import your component inside of it.